logic/overworld.py

```diff
@@ -529,8 +529,9 @@ def __init__(self, options, world_setup, r):
             self._addEntranceRequirement("d8", OR(BOMB, AND(OCARINA, SONG3))) # bomb trigger the head and walk trough, or play the ocarina song 3 and walk through
 
         if options.logic == 'hell':
-            dream_hut_right.connect(dream_hut, POWER_BRACELET) # grab the wall twice to control mimics without moving, and make them clip the wall to pass safely
-            swamp.connect(forest_toadstool, SHIELD) # damage boost from toadstool area across the pit.
+            dream_hut_right.connect(dream_hut, None) # alternate diagonal movement with orthogonal movement to control the mimics. Get them clipped into the walls to walk past
+            swamp.connect(forest_toadstool, None) # damage boost from toadstool area across the pit
```

Comment:
Does this work both ways? I've seen it being done from toadstool to swamp, but not the other way around.

Reply:
to get from swamp area to toadstool, you generally would use a pit buffer instead, although a damage boost would work too. The pit buffer from swamp to toadstool area is already added to glitch logic 
            swamp.connect(forest_toadstool, None, one_way=True) # villa buffer from top (swamp phonebooth area) to bottom (toadstool area)